mysql-test/suite/rocksdb/t/fast_prefix_index_fetch.test

```diff
@@ -0,0 +1,97 @@
+--source include/have_rocksdb.inc
+
+--disable_warnings
+DROP TABLE IF EXISTS t1;
+--enable_warnings
+
+CREATE TABLE t1 (
+  id int,
+  fake_id int,
+  bigfield varchar(4096),
+  PRIMARY KEY (id),
+  KEY (bigfield(32)),
+  KEY (fake_id, bigfield(32))
+) ENGINE=rocksdb;
+
+INSERT INTO t1 values (1, 1001, repeat('a', 1)),
+                      (8, 1008, repeat('b', 8)),
+                      (24, 1024, repeat('c', 24)),
+                      (31, 1031, repeat('d', 31)),
+                      (32, 1032, repeat('x', 32)),
+                      (33, 1033, repeat('y', 33)),
+                      (128, 1128, repeat('z', 128));
+
+SELECT * FROM t1;
+
+# After each test case, assert count of covered SK lookups is expected value.
+let $count_statement = show status like 'rocksdb_covered_secondary_key_lookups';
+
+--disable_query_log
+
+--echo # Baseline sanity check: 0, 0.
+--let $base_count = query_get_value($count_statement, Value, 1)
+select "no-op query";
+--let $count = query_get_value($count_statement, Value, 1)
+eval select $count - $base_count into @covered_lookups_count;
+select @covered_lookups_count = 0 as covered_lookups_count_matched;
+
+--echo # Eligible for optimization.
+--let $base_count = query_get_value($count_statement, Value, 1)
+select id, bigfield from t1 where bigfield = repeat('d', 31);
```

Comment:
Should also add in force index() to ensure the query plan selected uses this index. Depending on the run, it's possible it might do a full table scan and cause the test to be inconsistent.